Hyperlint Automation: Meta Description Fixes (#19738)
cloudflare/cloudflare-docs: src/content/docs/rules/index.mdx

pcx_content_type: overview
sidebar:
  order: 1

> description: Explore Cloudflare Rules to adjust requests and responses, configure settings, and trigger actions for specific requests. Learn via API, dashboard, or Terraform.

Reviewer: ```suggestion
description: Use Cloudflare Rules to adjust requests and responses, configure settings, and trigger actions for specific requests.
```

Reviewer: @bllchmbrs FYI - The second proposed sentence for the meta description wasn't too relevant.

Reviewer 2: Got it, so it was just a bit more information than you wanted?

Reviewer: Yes, or at least I didn't see it as something important we'd need to mention in the meta description. Many different (child) sections include API, dashboard, and Terraform instructions... so it's not something I'd highlight here.
